LayoutTests/imported/w3c/web-platform-tests/css/css-sizing/contain-intrinsic-size/auto-003-expected.txt

```diff
@@ -1,3 +1,3 @@
 
-FAIL contain-intrinsic-size: auto assert_equals: expected 1 but got 0
+FAIL contain-intrinsic-size: auto assert_equals: expected 50 but got 1
```

Comment:
`contain-intrinsic-size: auto && <length>` is not supported yet, it depends on `content-visibility`. https://bugs.webkit.org/show_bug.cgi?id=239450